src/App.tsx

```diff
@@ -94,7 +94,9 @@ const App = () => {
 		flowId === 'saml-config' ||
 		flowId === 'sso-config';
 
-	const containerClasses = clsx('descope-base-container', {
+	const shadow = urlParams.get('shadow') !== 'false';
```

Comment:
in addition to Nir's comment, I think we consider how to automate detect this and not asking developers to provide

Reply:
But it's not necessarily just mobile right, people generally may want to remove background shadow but it's personal preference. So no way to detect that.